retriever/pyproject.toml

```diff
@@ -7,17 +7,17 @@ readme = "README.md"
 package-mode = false
 
 [tool.poetry.dependencies]
-python = "^3.11"
-fastapi = "^0.108.0"
+python = ">=3.11,<3.13"
+fastapi = ">=0.109.2,<0.110.0"
 python-dotenv = "^1.0.0"
 confluent-kafka = "^2.3.0"
-uvicorn = "^0.25.0"
-pgvector = "^0.2.4"
+uvicorn = ">=0.27.0.post1,<0.28.0"
```

Comment:
Why is uvicorn required? All data should flow from API to Retriever through Kafka Qs

Reply:
It was already present, but threw package conflict for r2r, so updated it. Now removed